articles/using-ECS-to-deploy-a-docker-app-to-AWS/index.md

```diff
@@ -0,0 +1,222 @@
+### Introduction
+In this article, we will understand how to deploy a Docker-based application to Amazon Web pages using Elastic Container Service. Flask is a small python framework for building web applications.
```

Comment:
Amazon Web pages? Isn't is Amazon Web Services?

Also, in the line `Flask is a small python framework`, remove `small`.

Explain a little more about Flask framework and how it is related to this article.